Added description to add simulator in Xcode version 14.0 or  greater (#3671)
facebook/react-native-website: docs/_getting-started-macos-ios.md

To install a simulator, open <strong>Xcode > Preferences...</strong> and select the <strong>Components</strong> tab. Select a simulator with the corresponding version of iOS you wish to use.

> If you are using Xcode version 14.0 or greater than to install a simulator, open <strong>Xcode > Settin > Platforms</strong> tab. Click + icon to add more simulator. 

Reviewer: ```suggestion
If you are using Xcode version 14.0 or greater than to install a simulator, open <strong>Xcode > Settings > Platforms</strong> tab. Click + icon to add more simulator. 
```

> If you are using Xcode version 14.0 or greater than to install a simulator, open <strong>Xcode > Settin > Platforms</strong> tab. Click + icon to add more simulator. 

Reviewer 2: ```suggestion
If you are using Xcode version 14.0 or greater than to install a simulator, open **Xcode > Settings > Platforms** tab, then click "+" icon and select **iOS…** option. 
```

Few more tweaks for the sentence and switch to Markdown formatting.
